batchflow/models/tf/base.py

```diff
@@ -1031,6 +1032,20 @@ def predict(self, fetches=None, feed_dict=None, **kwargs):
             output = self.session.run(_fetches, _feed_dict)
         return self._fill_output(output, _fetches)
 
+    def _conversion_to_name(self, graph_item):
+        if isinstance(graph_item, dict):
+            for key in graph_item.keys():
+                graph_item.update({key: list(map(self._conversion_to_name, graph_item[key]))})
```

Comment:
```suggestion
                return {k: self._to_name(graph_item[k]) for k in graph_item)
```
Try not to change args implicitly